plugins/node/plugin.zsh

```diff
@@ -0,0 +1,18 @@
+# Color definitions
+GEOMETRY_COLOR_NODE_NPM_VERISON=${GEOMETRY_COLOR_NODE_NPM_VERISON:-black}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_NODE_NPM_VERISON=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
+GEOMETRY_NODE_NPM_VERISON=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERISON GEOMETRY_SYMBOL_NODE_NPM_VERISON) 
+
+geometry_prompt_node_setup() 
+
+geometry_prompt_node_render(){
+	local node_path="$echo "$(which node)" -v"
+	local npm_path="$echo "$(which npm)" --version"
+	GEOMETRY_NODE_VERSION=`(eval ${node_path})`
```

Comment:
There are a couple of issues here but my suggestion is to keep it simple:

1- Check node/npm are available
2- Display version info

For 1) zsh has a neat way to do it:

`$+commands[node]` (or for `npm`: `$+commands[npm]`).


You should check if there is no node/npm command available you should return an empty string:

    if (( ! $+commands[node] )); then
        return "";
    fi


For 2) you can simply do:

    GEOMETRY_NODE_VERSION="$(node -v)"